src/widget/wtrackmenu.cpp

```diff
@@ -153,7 +314,6 @@ void WTrackMenu::createActions() {
     }
 
     if (featureIsEnabled(Feature::LoadTo)) {
-        m_pAddToPreviewDeck = new QAction(tr("Preview Deck"), m_pLoadToMenu);
```

Comment:
Why was this removed? It seems unrelated to this PR.

Reply:
Probably caused by resolving a merge conflict? Damn!

Reply:
Fixed.